### PR TITLE
##4077208 UI feedbacks

### PR DESCRIPTION
**Pull request by nwashangai**
#### What does this PR do?
implement UI feedbacks
#### Description of Task to be completed?
* modify login button
* re-style background
* add categories of food
* pad input fields and element in modal
* change add new element to button and make color blue
* pad user order table cells to give texts space
* move food image upload button to the bottom of the form
#### How should this be manually tested?
* Not applicable
#### Any background context you want to provide?
* None
#### What are the relevant pivotal tracker stories?
#160512538
#160522121
#160528669
#160531883
#160532118
#160533269
#160533657
#### Screenshots (if appropriate)
* None
image
image
image

#### Questions:
* None

### CODE REVIEW

File: .travis.yml
Comment (dogoyaro) [inline .travis.yml:1]: Why have you deleted this file?

File: .gitignore
Comment (dogoyaro) [inline .gitignore:1]: Why have you deleted this file?

File: .eslintrc.json
Comment (dogoyaro) [inline .eslintrc.json:1]: Why have you deleted this file?

File: .babelrc
Comment (dogoyaro) [inline .babelrc:1]: Why have you deleted this file?

File: UI/admin.html
Comment (dogoyaro) [inline UI/admin.html]: I thought we agreed that an Avatar is not a good name to represent this field?

**Comment by dogoyaro** [comment]
You have deleted a lot of files in this PR, It'd be nice to know why

**Comment by dogoyaro** [comment]
You have to take another look at the header of your page. It should be simpler. Consider removing the image on top, moving the nav-bar to the top, and adding the name of the app to the black nav-bar.

**Review by dogoyaro — CHANGES_REQUESTED** [review]

File: .travis.yml
Comment (nwashangai) [reply .travis.yml:1]: When I checked out to my UI feedback branch from the develop branch which has all those files from my visual studio code, I noticed those API files where still present but I ignored them since they were marked as Untracked, but I realized later after pushing the code to the feedback branch that they where pushed as well, so I had to remove them to keep the UI code clean

File: .gitignore
Comment (nwashangai) [reply .gitignore:1]: Same as above

**Comment by nwashangai** [comment]
For the navbar, How about reducing the header image at the index, cos I noticed  the image is larger on the index page and it might look cool if reduced to a point that user can easily identify the navbar the moment the user opens the page

File: .travis.yml
Comment (dogoyaro) [reply .travis.yml:1]: gotcha.

**Comment by dogoyaro** [comment]
I think I might have an opinion if I can see how it looks.

File: UI/admin.html
Comment (dogoyaro) [reply UI/admin.html]: @nwashangai. Still waiting on the response for this. Are you fixing this here? would love to approve this PR if you are.

**Review by dogoyaro — APPROVED** [review]

**Pull request merged**
